Source/MethodCall.cs

```diff
@@ -129,6 +123,11 @@ public MethodCall(Mock mock, Condition condition, Expression originalExpression,
 						throw new NotSupportedException(Resources.OutExpressionMustBeConstantValue);
 					}
 
+					if (outValues == null)
+					{
+						outValues = new List<KeyValuePair<int, object>>();
+					}
```

Comment:
`outValues` is no longer guaranteed to be pre-allocated, so we need to check here. This won't hurt a lot, since it's only checked once per `out` parameter (of which there usually aren't many). Also, we're in a ctor, so no need to worry about thread safety.